### PR TITLE
Comments popup

### PR DESCRIPTION
**Pull request by krisbobo**
Milestone: Comments Popup
*The following changes are being introduced:*
- When the popup loads, the web app retrieves data from the API and shows details about the selected item.
- Created a popup with the selected item's details.
- Add an event to a button prepared by a teammate that will open your popup.
**This PR resolves issue #14**

### CODE REVIEW

File: src/modules/popullateMovie.js [hunk 1]
```diff
@@ -34,4 +35,33 @@ export const showMovies= async () => {
...
+                </div>
+            </div>
+            `;
+            const closeBtn = document.querySelector('.close');
+            closeBtn.addEventListener('click', () => {
+                window.location.reload();
+            });
+            })
+        });
+    }
```
Comment (akos29) [inline src/modules/popullateMovie.js]: I kindly recommend making sure the popup is targeting the right item when it is loading. The requirement specifies the webapp shall retrieve data from the selected API and shows details about the selected item. In your case, it is showing the last movie regardless of clicking on any item on the list.

File: src/style.css [hunk 2]
```diff
@@ -138,6 +139,54 @@ header {
...
+  display: var(--flex);
+  justify-content: space-between;
+  flex-wrap: wrap;
+}
+
+.popup-body img {
+  width: max-content;
+  padding: 1rem;
+}
+
```
Comment (akos29) [inline src/style.css]: What you have done so far is good 😉 . However, you still need to make your design looks similar to the requirement. 
image
Your work :
image

**Review by akos29 — CHANGES_REQUESTED** [review]
### Change required ♻️

You have put a great effort to reach here 👏  

- No linter error ✔️ 
- Descriptive Commits ✔️ 
- Popup opens nicely ✔️ 

### Suggestions

 - [ ] Kindly you may consider the popup opens the right element
 - [ ] Kindly make you design similar to the requirement

File: src/style.css (hunk 2 above)
Comment (krisbobo) [reply src/style.css]: 👍 Will get right on it

**Review by akos29 — APPROVED** [review]
### **STATUS APPROVED** 🥳🥳🥳

Hi @krisbobo 

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

ThumbsUpWellDoneGIF (2)

**Pull request merged**